README.md

```diff
@@ -43,3 +44,15 @@
 This project is completely Open source. Any willing to contribute can take up an issue or raise a new and work on it after getting assigned.
 
 ---
+## Contributors :v:
```

Comment:
Please add it as the last section of readme file.

Reply:
Hey it's added in the last section of README.md, in these two screenshots above one is of the below section where that section is....and second image is of contents at the top of readme file......you may see it on my commited branch

Reply:
apologies for that